### PR TITLE
800. Add sbom section to deploy configure/review

### PR DESCRIPTION
**Pull request by mike-winberry**
## Description
#800 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Other (security config, docs update, etc)

## Checklist before merging

- [x] Test, docs, adr added or updated as needed
- [ ] [Contributor Guide Steps](https://github.com/defenseunicorns/zarf/blob/main/CONTRIBUTING.md#developer-workflow) followed

### CODE REVIEW

**Comment by mike-winberry** [comment]
https://github.com/defenseunicorns/zarf/assets/86802655/935dd505-7974-4a65-b296-456fa59d02d0

**Comment by Noxsios** [comment]
A few tweaks:

- Change the triangle "warning" icon on the SBOM section to be amber
- Change the second sentence to be `You can view them here:` and then place the link on a new line

**Comment by mike-winberry** [comment]
@Racer159 @Madeline-UX 
Made the changes we discussed should be ready to go for full review. 
https://github.com/defenseunicorns/zarf/assets/86802655/8a7a9253-f2f7-47d0-80e8-dac31c4007a5

**Review by Madeline-UX — DISMISSED** [review]

File: src/internal/api/packages/sbom.go [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// Extracts the SBOM from the package and returns the path to the SBOM
+func extractSBOM(escapedPath string) (sbom types.APIPackageSBOM, err error) {
+	const sbomDir = "zarf-sbom"
+	const SBOM = "sboms.tar"
```
Comment (Racer159) [inline src/internal/api/packages/sbom.go:62]:
```suggestion
```

`zarf-sbom` is not used and `SBOM` could be placed next to `config.ZarfSBOMDir` as `config.ZarfSBOMTar` replacing the other places the string literal `sboms.tar` is used (`src/pkg/packager/inspect.go`, `src/pkg/packager/common.go`, and `src/cmd/tools/archiver.go`)

**Review by Racer159 — CHANGES_REQUESTED** [review]

**Review by Racer159 — APPROVED** [review]

**Review by Noxsios — APPROVED** [review]
lgtm

**Pull request merged**